src/components/AddressSearch.js

```diff
@@ -82,14 +83,30 @@ const AddressSearch = (props) => {
         }
     };
 
+    const checkPressingTab = (event) => {
+        if (event.key !== 'Tab') {
```

Comment:
hmm, I don't think this needs to be it's own variable. Why not do the following? 
```Javascript
onKeyPress: (event) => {
    if (event.key !== 'Tab' || isSelected) {
        return;
    }
    googlePlacesRef.current.setAddressText('');
},
```

Reply:
We tried similar way, but with "&&" operator instead "||", which led to the fact that only the last entered character would fit in the text field. Thank you.